test/sandbox.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
sandbox and sandbox-inside doesn't play well with the other html files in this folder. It's repeating another test and the intent is implicit only. After this PR one would look and ask why this is here. The same can be asked when reading the Gruntfile.

naming is hard, be we should try again w/ this one. Maybe sessionstorage-through-iframe.html is long but good enough, I'm bad with names too.